doc/roadmap.md

```diff
@@ -75,6 +74,7 @@ you wish to get involved in the implementation or influence the roadmap
 
 The following list contains all features that were in the roadmap and have already been implemented.
 
+- MQTT Retain flag ([#4388](https://github.com/telefonicaid/fiware-orion/issues/4388)) ([3.11.0](https://github.com/telefonicaid/fiware-orion/releases/tag/3.11.0))
```

Comment:
3.11.0 Pending to be released

Reply:
> Should we hold this PR until 3.11.0 gets released?

I think it really does not matter, but we can wait till having the real tag

Reply:
Otherwise, the included link https://github.com/telefonicaid/fiware-orion/releases/tag/3.11.0 is not going to work...

Reply:
Orion 3.11.0 has ben released. This PR can now be merged.